.github/workflows/checks.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       - name: Compile all code with fatal warnings for Java 11, Scala 2.12 and Scala 2.13
         # Run locally with: sbt 'clean ; +Test/compile ; +It/compile'
-        run: sbt "; Test/compile; It/compile"
```

Comment:
This wasn't even properly cross compiling, i.e. it was only compiling for the default Scala version and not all of `CrossScalaVersions`